test/test_using_plams/test_opt_orca.py

```diff
@@ -67,7 +64,8 @@ def test_methanol_opt_orca():
 
     coords = collapse([a.coords for a in mol_opt.atoms])
 
-    assert abs(sum(zipWith(operator.sub)(coords)(expected_coords))) < 1e-7
+    assert abs(sum(real - expected for (real, expected)
```

Comment:
You could consider using [``math.isclose()``](https://docs.python.org/3/library/math.html#math.isclose) here. Note that the ``1e-7`` would be equivalent here to the ``abs_tol`` argument, not ``rel_tol``.

Reply:
:+1: